lib/Myriad/RPC/Implementation/Redis.pm

```diff
@@ -84,16 +86,43 @@ async method stop () {
 async method create_group ($rpc) {
     unless ($rpc->{group}) {
         await $self->redis->create_group($rpc->{stream}, $self->group_name);
+        await $self->process_pending($rpc);
         $rpc->{group} = 1;
     }
 }
 
+async method process_pending ($rpc) {
+    my $pending_info = await $self->redis->pending_messages_info($rpc->{stream}, $self->group_name);
+    while ($pending_info->[0] > 0) {
```

Comment:
I think a big backlog of pending messages is a concern here, what should we do? 
- mix pending and new messages randomly? 
- process pending messages when there are no new messages.
- add an option for the developer to drop processing if the backlog is more than a certain threshold?

Reply:
If there's a big backlog of pending messages, then we process those first.